diff --git a/sites/all/modules/registration/registration.module b/sites/all/modules/registration/registration.module
index b7b20d9..4913b06 100644
--- a/sites/all/modules/registration/registration.module
+++ b/sites/all/modules/registration/registration.module
@@ -958,6 +958,25 @@ function registration_send_broadcast($entity_type, $entity_id, $subject, $messag
     $registrations = registration_load_multiple(
       array_keys($entities['registration'])
     );
+    
+    
+    
+    // Give other modules a chance to alter registrations (for example, did 
+    // someone opt-out of the reminder?
+    
+    // Invoke hook_registration_broadcast_registrations_alter(), 
+    // hook_registration_broadcast_registrations_ENTITY_TYPE_alter() and
+    // hook_registration_broadcast_registrations_ENTITY_TYPE_ENTITY_ID_alter()
+    $hooks = array('registration_broadcast_registrations');
+    $hooks[] = 'registration_broadcast_registrations_' . $entity_type;
+    $hooks[] = 'registration_broadcast_registrations_' . $entity_type.'_'.$entity_id;
+    $context = array(
+      'entity_type' => $entity_type,
+      'entity_id' => $entity_id,
+    );
+    drupal_alter($hooks, $registrations, $context);
+    
+    
 
     // send the email to each registrant and communicate results
     $success_count = 0;
